src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -744,6 +744,7 @@ void SpatialPooler::updateDutyCyclesHelper_(vector<Real> &dutyCycles,
 
 
 void SpatialPooler::updateBoostFactors_() {
+  if(boostStrength_ < htm::Epsilon) return; //skip for disabled boosting
```

Comment:
This won't work because `SpatialPooler::BOOSTING_LOG < htm::Epsilon` and so it will skip log-boosting.

Reply:
I could use fabs(), or use exact match str_ == DISABLED